CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+
+## [29.5.5] - 2020-08-21
+- Add extension data schema resolvers to data schema parser to parse schema from both /extensions and /pegasus jar entry in the same jar file.
```

Comment:
I think the description is not accurate. 
Add a constructor for DataSchemaParser, which is able to pass ExtensionsDataSchemaResolver to the DataSchemaParser to parse schemas from both extensions and pegasus directories.